ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/response/HateoasLink.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Shall we move this to `hateoas` sub module?

Reply:
Fixed with [e458073d607a52d29e51db349f677810a0e85b8a](https://github.com/ballerina-platform/openapi-tools/pull/1589/commits/e458073d607a52d29e51db349f677810a0e85b8a)